driver/src/io_backend/ext/xaie_io_common.c

```diff
@@ -392,7 +392,9 @@ static AieRC _XAie_RequestBroadcastChannelRscCommon(XAie_DevInst *DevInst,
 AieRC _XAie_RequestRscCommon(XAie_DevInst *DevInst, XAie_BackendTilesRsc *Args)
 {
 	AieRC RC;
-	u32 RscArrPerTile[Args->NumRscPerTile];
+	u32 *RscArrPerTile = malloc(sizeof(u32) * Args->NumRscPerTile);
```

Comment:
alloca() instead, otherwise you end up with a memory leak?

Reply:
Actually i don't know why this change was made at all - the deleted code is a [VLA](https://gcc.gnu.org/onlinedocs/gcc/Variable-Length.html) which is a C99 feature, which the corresponding PR in mlir-aie [specifies](https://github.com/Xilinx/mlir-aie/pull/1068/files?file-filters%5B%5D=.cmake&show-viewed-files=true#diff-403504e9fb1f718580537ce743cd700e5e4fdc5987a7dc93b5271679d5d2ca4a).

Reply:
Answer: MSVC doesn't support VLAs okay

Reply:
Solution: `free`